src/languages/bash.js

```diff
@@ -38,6 +38,18 @@ export default function(hljs) {
     end: /\)/,
     contains: [ hljs.BACKSLASH_ESCAPE ]
   };
+  const COMMENT = hljs.inherit(
+    hljs.COMMENT(undefined, undefined),
```

Comment:
Wouldn't `hljs.COMMENT()` be equivalent - since unpaused arguments default to undefined?

Reply:
Absolutely. I'm very used to strict typescript where that would be forbidden 😄
Fixed!